### DISCUSSION TITLE
Frequency Manager display plus Omnirig

### DISCUSSION THREAD

**Discussion by Streety66** [AlexandreRouma/SDRPlusPlus#1095] [Ideas]
Hi Alexandre - new to SDR++, I have switched from SDR# and SDRUno as I have Airspy HF+, Airspy Mini, 2 x RSP1A, RTL-SDR V3 and a Nooelec Smartee to play with. Currently only using it on Windows as I use DSDPlus FastLane. But I do have some RPI3 and 4 to play with.

Two things from me that would help if it's possible to look at, please.

1. Frequency Manager, would it be possible to display on the waterfall only those frequencies stored in the selected group. I have 2 sites with frequencies stored in 2 groups. Currently both sites are displaying on the waterfall, even though I have only a single group selected.
2. I have paid for RRUK Linker which talks to both SDR# and SDRUno via Omnirig - I contacted RRUK to see if they can implement a solution but they stated it would be easier if SDR++ interfaces with Omnirig.

I am very impressed with SDR++, thank you for your hard work.

Paul.

**Comment by AlexandreRouma** [comment]
hi,

1) Switching displayed group automatically when switching list is not possible BUT you can select only one group to be displayed by clicking on "select displayed lists" and selecting the one you want only.

2) Onmirig support is not planned as SDR++ already supports the much more common Hamlib NET rigctl interface. Most software (wsjtx, qpredict, etc) already use that interface. All RRUK would have to do is support that interface which is quite simple.

best regards

**Reply by Streety66** [reply]
Goodness me, that was a quick reply, thanks Alexandre.

1. Brilliant, that is exactly what I was after, thank you for the pointer.
2. Thanks, I shall go back to RRUK and suggest this.

All the very best to you.
Paul.
